h2o-core/src/main/java/water/api/CapabilitiesHandler.java

```diff
@@ -18,7 +18,7 @@
     return entries;
   }
 
-  private ArrayList<CapabilityEntryV3> getRestAPIExtensionEntries(){
+  protected ArrayList<CapabilityEntryV3> getRestAPIExtensionEntries(){
```

Comment:
why is this needed?

Reply:
Will put back to private. I am using getRestApiExtensions() instead and must've forgotten to tidy this.